pkg/provider/kubernetes/ingress-nginx/kubernetes.go

```diff
@@ -934,6 +936,22 @@ func applyCORSConfiguration(routerName string, ingressConfig ingressConfig, rt *
 	rt.Middlewares = append(rt.Middlewares, corsMiddlewareName)
 }
 
+func applyWhitelistSourceRangeConfiguration(routerName string, ingressConfig ingressConfig, rt *dynamic.Router, conf *dynamic.Configuration) {
+	var whitelistSourceRange = ptr.Deref(ingressConfig.WhitelistSourceRange, "")
+	if whitelistSourceRange == "" {
+		return
+	}
+
+	WhitelistSourceRangeMiddlewareName := routerName + "-whitelist-source-range"
```

Comment:
nit: as this is a local variable, better to name it `whitelistSourceRangeMiddlewareName`, without capital W.